Client/mods/deathmatch/logic/CClientBuilding.cpp

```diff
@@ -132,6 +132,7 @@ void CClientBuilding::Create()
     ConvertZXYEulersToQuaternion(m_vRot, vRot4D);
 
     m_pBuilding = g_pGame->GetPools()->GetBuildingsPool().AddBuilding(this, m_usModelId, &m_vPos, &vRot4D, m_interior);
+    m_pBuilding->SetBackfaceCulled(!m_bDoubleSided);
```

Comment:
Is there any chance of `m_pBuilding` being `nullptr`?

Reply:
There is an edge case, but i prevent it on API level

Reply:
According to the source code, it can return `nullptr` when `if (!HasFreeBuildingSlot())`

Reply:
Ok, but I'll prefer a crash in this case.